src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -596,11 +609,32 @@ private void SetMdpScope(IList<CommPkg> commPkgs)
         }
 
         private void RemoveOldCommPkgs(IList<CommPkg> commPkgs)
-            => _commPkgs.RemoveAll(x => !commPkgs.Select(y => y.CommPkgNo).Contains(x.CommPkgNo));
+        {
+            var mcPkgsToRemoves = _commPkgs.Where(m => commPkgs.All(m2 => m2.CommPkgNo != m.CommPkgNo)).ToList();
```

Comment:
typo.

Reply:
(and if you care, Any() is more performant, but I dont think it matters for this size )